README.md

```diff
@@ -1 +1,56 @@
-# school-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# ruby-school-library
+
+> School library app.
+
+## Built With
+
+- https://img.shields.io/badge/Ruby-CC342D?style=for-the-badge&logo=ruby&logoColor=white
+
+
+## Getting Started
+### Prerequisites
+
+- `ruby`:  `^3.0.1`
+
+## Setup
+
+You can run it on your local machine by cloning the project:
+- Click on the green "Code" button above.
+- Copy the link.
+- On your Terminal run `git clone $<LINK>` replacing `$<LINK>` with the link you just copied.
+- cd to the folder where you cloned the repo.
+- Now you can run the commnads below as needed.
+
+## Usage
+
+In the project directory you can run the following command to execute the code:
+
+`> ruby main.rb`
+
+## Authors
+
+👤 Mohamed El-Edeeb
+
+- [GitHub](https://github.com/eng-mohamed-eldeeb)
+- [LinkedIn](https://www.linkedin.com/in/eng-mohamed-eldeeb/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse's Curriculum project
+- Thanks to all code reviewers
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As stated previously by the reviewer, consider using [a new README template](https://github.com/microverseinc/readme-template) not the old one and make sure to include all main headers. Also, consider customizing it for your project.